project/profiles/capacity4more/behat/features/group_content_permissions.feature

```diff
@@ -14,109 +14,859 @@ Feature: Test creation of the content permissions.
+      | mariecurie  |  /deleted/node/add/wiki-page |
+
   @api
-  Scenario: There should be no access to 'create photoalbum' global form.
-    Given I am logged in as user "isaacnewton"
-    When  I go to "node/add/photoalbum"
+  Scenario Outline: As non member, member, group admin and group owner, access create content form, with purl prefix of deleted public group.
+    Given  I am logged in as user "<user>"
+    When  I go to "<path>"
     Then  I should not have access to the page
 
+    Examples:
+      | user        | path                 |
+      | badhairday  | /deleted/node/add/photoalbum |
+      | isaacnewton | /deleted/node/add/photoalbum |
+      | turing      | /deleted/node/add/photoalbum |
+      | alfrednobel | /deleted/node/add/photoalbum |
+      | badhairday  | /deleted/node/add/photo |
+      | isaacnewton | /deleted/node/add/photo |
+      | turing      | /deleted/node/add/photo |
+      | alfrednobel | /deleted/node/add/photo |
+      | badhairday  | /deleted/node/add/event |
+      | isaacnewton | /deleted/node/add/event |
+      | turing      | /deleted/node/add/event |
+      | alfrednobel | /deleted/node/add/event |
+      | badhairday  | /deleted/node/add/document |
+      | isaacnewton | /deleted/node/add/document |
+      | turing      | /deleted/node/add/document |
+      | alfrednobel | /deleted/node/add/document |
+      | badhairday  | /deleted/node/add/discussion |
+      | isaacnewton | /deleted/node/add/discussion |
+      | turing      | /deleted/node/add/discussion |
+      | alfrednobel | /deleted/node/add/discussion |
+      | badhairday  | /deleted/node/add/wiki-page |
+      | isaacnewton | /deleted/node/add/wiki-page |
+      | turing      | /deleted/node/add/wiki-page |
+      | alfrednobel | /deleted/node/add/wiki-page |
+
   @api
-  Scenario: Group non member should not have access to group 'create photoalbum' form.
-    Given I am logged in as user "badhairday"
-    When  I go to "movie-corner/node/add/photoalbum"
-    Then  I should not have access to the page
+  Scenario: Set "Deleted group" group state to 'Restricted'. Anonymous user asked to log in.
+    Given I am logged in as user "mariecurie"
+    When  I change access of group "Deleted group" to "Restricted"
+    Then  I am an anonymous user
+    And   I go to "deleted"
+    And   I should see "Please log in"
+
+  @api
+  Scenario Outline: As site admin, access create content form, with purl prefix of deleted restricted group.
+    Given  I am logged in as user "<user>"
+    When   I go to "<path>"
+    Then   I should have access to the page
+
+    Examples:
+      | user        | path                 |
+      | mariecurie  | /deleted/node/add/photoalbum |
+      | mariecurie  | /deleted/node/add/photo |
+      | mariecurie  | /deleted/node/add/event |
+      | mariecurie  | /deleted/node/add/document |
+      | mariecurie  | /deleted/node/add/discussion |
+      | mariecurie  | /deleted/node/add/wiki-page |
+
```

Comment:
same comment as above, you can remove this scenario and add the SA to the scenario described on line 780